chain/paseo/defaults.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2021 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package paseo
+
+import (
+	cfg "github.com/ChainSafe/gossamer/config"
+)
+
+var (
+	// defaultName Default node name
+	defaultName = "Paseo"
+	// defaultID Default chain ID
+	defaultID = "paseo"
+	// defaultBasePath Default node base directory path
+	defaultBasePath = "~/.gossamer/paseo"
```

Comment:
This should be `$XDG_DATA_HOME`, I'll create a new issue

https://github.com/ChainSafe/gossamer/issues/3678

Reply:
Gonna have to make a small change for this #3211

Reply:
Thank you!